src/components/ListState.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+  <v-row class="justify-start align-center">
+    <v-icon>drag_indicator</v-icon>
+    <div id="icon-state">
+      <icon-state :passedIcon="item.icon" @update:icon="choosenIcon($event)" class="mr-2 ml-2"></icon-state>
```

Comment:
I think if you use `icon` instead of `passedIcon`, you can use `icon.sync=icon` shortcut.